spec/variable_spec.rb

```diff
@@ -12,6 +12,16 @@
     end
   end
 
+  context "symbol" do
+    let(:var) { KingKonf::Variable.new(name: "codec", type: :symbol) }
+    it "casts strings" do
+      expect(var.cast("gzip")).to eq :gzip
+    end
+    it "doesn't cast integers" do
+      expect(var.cast(90)).to eq 90
+    end
+
+  end
```

Comment:
Can you make the whitespace similar to the surrounding code?

Reply:
Done.